util.inc

```diff
@@ -28,13 +28,39 @@
 		return $result;
 	}
 
+	// Do a simple query against the named database.
+	// WARNING: Do not allow any user input into the statement!
+	// Recommend using dbBoundQuery() instead if user input is necessary.
 	function dbQuery($dbname, $sql){
 		include("dbconfig.inc");
 		$con = mysqli_connect($dbhost, $dbuser, $dbpass);
-		if (!$con)	{  die('Could not connect: ' . mysql_error()); }
+		if (!$con)	{  die('Could not connect: ' . mysqli_connect_error()); }
```

Comment:
I think we've configured things so that basically any php error will just be printed to the user (I know I have seen these to the user in testing).  It might go to the web server error log too, but I don't see anything obviously like this on my dev machine and I'm pretty sure I hit it once or twice.

I do know that mysql_error is a function that doesn't actually exist, so causes its own error :)

Reply:
Hah I was wondering!